release/ray_release/configs/global_config.py

```diff
@@ -0,0 +1,45 @@
+import os
+import yaml
+
+
+class GlobalConfig:
```

Comment:
maybe just use a `TypedDict`, so that you do not need to write all those getters?

Reply:
lol, I tried it but TypedDict does not allow static variable so I cannot create a singleton instance of it